pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# Description
+
+-   Explain the purpose of this pr
+
+# Testing instructions
+
+-   Add testing instructions if required
+
+# Type of change
+
+-   [ ] Bug fix (non-breaking change which fixes an issue)
+-   [ ] New feature (non-breaking change which adds functionality)
+-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
Maybe keep the docs option in here in case there are times we just want to update the documentation

Reply:
sure will put it back in